gooddata-java/src/main/java/com/gooddata/sdk/service/project/ProjectService.java

```diff
@@ -62,26 +89,13 @@ public ProjectService(final RestTemplate restTemplate, final AccountService acco
     }
 
     /**
-     * Get all projects current user has access to.
-     *
-     * @return collection of all projects current user has access to
-     * @throws com.gooddata.GoodDataException when projects can't be accessed
-     * @deprecated use {@link #listProjects()} or {@link #listProjects(Page)} instead.
-     * Deprecated since version 3.0.0. Will be removed in one of future versions.
```

Comment:
This was meant to keep deprecated in 3.0.0. Is there any problem with that?

Reply:
ahh my mistake, reverted in fixup